src/Utils.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import MetaProperty from "./MetaProperty";
+import { PathUtils } from "./PathUtils";
+import Constants from "./Constants";
+
+export class Utils {
+
+    /**
+     * Returns true if the editor is in Edit Mode
+     *
+     * @returns {Boolean} the result of the check of the Editor mode
+     */
+    public static isEditMode(): boolean {
+        const aemMode = PathUtils.getMetaPropertyValue(MetaProperty.WCM_MODE);
```

Comment:
i know you did not implement it, but `getMetaPropertyValue` does not fit to `PathUtils`

Reply:
Correct. If that is API, we'll have to deprecate when relocating. Not sure if we want to do it via the current PR. Maybe a GitHub issue open to the community?